OneCommunityGlobal/HighestGoodNetworkApp
# Fix/create tasks bug

# Description
This PR is related to the 3 Medium BUGS in the HGN Phase bugs. You can check more details about the bug at the [DOCS ](https://docs.google.com/document/d/1n5NIb3dbiSrIRuOG1ZCwt34pEP1poqddCv1guaQml88/edit), but this PR will change:
- The default option when creating a new task (now is Housing); 
![CreatingANewTaskOptionDefault](https://user-images.githubusercontent.com/100925414/209458963-dec1bfb0-112c-4344-8bf6-828afd309149.jpg)
- If a task has a MOTHER, It means, if a new task is being creating as SUB-TASK (above a task), the default classification will be the mother (but It can be change using the select);
- Correct the time registered by a volunteer in the correct classification;
-
## How to test:
-  Login as admin -> go to OtherLinks -> Projects -> WBS -> Add task 
- After you should add a new task and assign to a volunteer account that you have, and in the volunteer account, you should check if the time is being to the correct classification. (I made a video testing this just to show how I tested, It can be helpful also).
